### ISSUE TITLE
Something wrong in OSX package.

### ISSUE THREAD

**Issue by KepenJ** [Moeditor/Moeditor#6]
I used electron-packager to export osx package,but when I run i found the "Edit" menu inside were gone.
At the same icon is also a problem.
The following are my modifications to the file package.json

`  "scripts": {
    "start": "electron .",
    "package": "electron-packager ~/Desktop/Moeditor-master Moeditor --platform=mas --arch=x64 --out ~/Desktop/MoeTest --version 1.0.0 --overwrite --icon=./icons/Moeditor.svg"
  },`

**Comment by Menci** [comment]
The package command I used to package for OS X on OS X was

```
electron-packager . Moeditor --platform=darwin --arch=x64
```

I think you were using the wrong **`--platform`**, [electron-packager](https://github.com/electron-userland/electron-packager)'s readme tells we should use **`darwin`** instead of **`mas`** for OS X.

It seems the platform **mas** means 'Mac App Store'. I never used Mac App Store, and is that something like the Ubuntu's Software Center maintained by Apple? If so, I think it may be used to make a zipped package like .deb? The **darwin** option gave me a .app folder.

And about the icon, I don't know the `--icon` argument, so I packaged without that, and then modified `Info.plist` and tried to change `Electorn.icns` to `Moeditor.icns`. Then I googled how to make icns file. Finally I put my `.icns` file in that folder and the icon was showed.

I think `electron-packager` can't generate `.icns` icon. I made my icon with Inkscape and rendered that to .png with GIMP, but neither of them can export `.icns` format. Maybe you should use Apple's toolchain and way for the icon.

You said your "Edit" menu was gone. Did you noticed the console output (with running /MacOS/Executable) ? I set the menu in the main process, so if there's an error of menu, it may be logged to the console. But why there wasn't an exception.

Was everything in my build correct? Or try to `npm start` or build with my command, and test it.

**Comment by KepenJ** [comment]
I found the "Edit" problem  should be **low version** of  [electron-packager](https://github.com/electron-userland/electron-packager/blob/master/usage.txt) export related reasons . So,I remove `--version 1.0.0` to use the default version(1.2.5)  will don't cause problems.  

About `darwin ` and `mas` , I looked at the [electron-packager](https://github.com/electron-userland/electron-packager) source code found that if you do not set certificate and profile issues , two of them exported app content and format is the same.

And the icon,like you said it's a problem of format.
